.github/workflows/release-lane.yml

```
@@ -11,93 +11,184 @@ concurrency:
-      - name: Extract
-        run: |
-          tar -xzf "${TF_ARTIFACT_NAME}-${TF_SHA}.tgz"
-
-      - name: Deploy PROD
-        run: |
-          echo "Deploying ${TF_SHA} to PROD..."
+          host: ${{ secrets.PROD_HOST }}
+          username: deploy
+          key: ${{ secrets.PROD_SSH_KEY }}
+          script_stop: true
+          script: |
```

actionlint (1.7.11)

[error] property "build_once" is not defined in object type {promote_staging: {outputs: {}; result: string}}

(expression)